app/channels/map_channel.rb

```diff
@@ -56,4 +56,10 @@ def move_token(data)
       }
     )
   end
+
+  private
+
+  def render_token(token)
+    render_partial("tokens/token", locals: { token: token })
+  end
```

Comment:
Is this used anywhere? I saw the same code in https://github.com/jdbann/table/pull/46/files#diff-475bdc053fa7a9895b79ba5abf97474dR18-R23 and was wondering if maybe the channel should own the `broadcast_to` methods

Reply:
I originally had it in `MapChannel` but it needed to be a class level method, and that didn't seem great, so I moved it out and accidentally left this method.

That being said, I think maybe the benefit of it all being in `MapChannel` overrides that I've moved it there in https://github.com/jdbann/table/pull/46/commits/e22b1f0c8be1c25de9858f12b4ecb29e04c01dcd and we can always refactor if we think it'd be better elsewhere.